### PR TITLE
Introducing the redirect url (for generic user action)

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: ui/app/com/gu/recipeasy/controllers/Application.scala [hunk 1]
```diff
@@ -29,6 +29,27 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
     curatedRecipedEditor(recipe, editable = false, pageTopMessage = "", showSkipThisRecipeButton = false)
   }
 
+  def genericActionShouldRedirectToCreation(): Boolean = {
```
Comment (LATaylor-guardian) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: This can be private.

Comment (shtukas) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: `isShowCreation` Done.

**Comment by shtukas** [comment]
Done!

File: ui/app/com/gu/recipeasy/controllers/Application.scala [hunk 2]
```diff
@@ -29,6 +29,27 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
...
+    // TODO: write the logic to compute the correct answer here
+    true
+  }
+
+  def genericUserAction() = Action { implicit request =>
```
Comment (LATaylor-guardian) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: `displayView`?

**Comment by LATaylor-guardian** [comment]
👍  other than naming.

File: ui/app/com/gu/recipeasy/controllers/Application.scala (hunk 2 above)
Comment (shtukas) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: `displayView` Done as well.

File: ui/conf/routes [hunk 3]
```diff
@@ -5,9 +5,11 @@ GET        /healthcheck                                                   contro
 
 # User navigation
 GET        /                                                              controllers.Application.index
+
 GET        /recipe/curate                                                 controllers.Application.curateOneRecipeInNewStatus
 GET        /recipe/curate/:recipeId                                       controllers.Application.curateRecipe(recipeId)
 GET        /recipe/verify/:recipeId                                       controllers.Application.verifyRecipe(recipeId)
+GET        /recipe/redirect                                               controllers.Application.displayView
```
Comment (mchv) [inline ui/conf/routes]: `/recipe/curate-or-verify`

Comment (shtukas) [inline ui/conf/routes]: Done!

**Comment by shtukas** [comment]
@LATaylor-guardian I am now using url `/recipe/curate-or-verify` so I renamed the controller to `curateOrVerify` (wasn't very important but for consistency...)

**Pull request merged**